.gitbook/FAQ.md

```diff
@@ -43,7 +43,7 @@ Many wallets now allow applications to trigger a popup to switch between network
 
 \* [Universal gateway](https://gateway.boba.network).
 
-If your wallet does not support this feature, you will have to connect manually. The exact process for connecting your wallet to a Boba Ethereum network depends on the specific wallet software you are using. To get started on Boba/Ethereum, you can use the available [community RPC endpoint](https://docs.boba.network/developer-docs/network-eth).
+If your wallet does not support this feature, you will have to connect manually. The exact process for connecting your wallet to a Boba Ethereum network depends on the specific wallet software you are using. To get started on Boba/Ethereum, you can use the available [community RPC endpoint](https://docs.boba.network/developer/network-eth).
```

Comment:
Since the links are explicit and not relative, it's a bit hard to test.  But, I believe this should be to `developer/multichain/network-eth`?

Also, why is this link absolute when the others are relative?

Reply:
Updated to the relative link